workbench-core/environments/src/services/environmentService.ts

```diff
@@ -76,13 +77,10 @@ const defaultEnv: Environment = {
 };
 
 export class EnvironmentService {
-  private _aws: AwsService;
-  private _tableName: string;
+  private _dynamoDBService: DynamoDBService;
 
-  public constructor(constants: { TABLE_NAME: string }) {
-    const { TABLE_NAME } = constants;
-    this._tableName = TABLE_NAME;
-    this._aws = new AwsService({ region: process.env.AWS_REGION!, ddbTableName: TABLE_NAME });
+  public constructor(dynamoDBService: DynamoDBService) {
```

Comment:
With this change, we'll also need to update the way this service is being referenced in these:
`backendAPI`, `sagemakerNotebookEnvironmentLifecycleService`, `statusHandler`, `environmentLifecycleHelper`.

Reply:
🤦 thank you. rush build didn't fail which is really frustrating. maybe i forgot to run it after that change, not sure.